docs/UserGuide.md

```diff
@@ -534,37 +551,28 @@ Notor has keyboard shortcuts to give you added convenience and allow you to cont
 **In Pop-Up Windows**
 Navigate the buttons using the arrow keys. On Windows and Linux, select the button by pressing enter; on MacOS, press space.
 
-**Note**:
-- **TimeStamp** shortcut key types the current date and time in Note Window in this format `E, MMM dd yyyy HH:mm"`.
-- Example: `Fri., Oct. 22 2021 00:07`
-
 Shortcut Key            | Linux/Window                                   | MacOS
 ------------------------|------------------------------------------------|---------------------------------------
 **TimeStamp**           | `Ctrl + T`                                     | `Cmd + T`
 **Save**                | `Ctrl + S`                                     | `Cmd + S`
 **Quit without Saving** | `Ctrl + W`                                     | `Cmd + W`
 **Save and Quit**       | `Ctrl + Q`                                     | `Cmd + Q`
 
+<div markdown="block" class="alert alert-info">
+**Notes on Timestamp:**<br>
+- **TimeStamp** shortcut key types the current date and time in Note Window in this format `E, MMM dd yyyy HH:mm"`.
+- Example: `Fri., Oct. 22 2021 00:07`
+</div>
 
-##<a name = "command-summary"></a> Command Summary
+<a name = "command-summary"></a> 
+## Command Summary
+<div markdown="block" class="alert alert-info">
 Round brackets `()` refer to COMPULSORY arguments.
 Square brackets `[]` refer to optional arguments.<p>
-*TODO: Make command action words below link to their entries above.*
+</div>
 
 ### Person
-**Note**:
-* For the **Create** and **List** commands, if you want to automatically add them to a group, please use the **List** command to make sure the `Group` you want to
-  use the command on is displayed before using them via `GROUP_INDEX`.
-* For the **Add** and **Remove** commands, please make sure that the `GROUP_NAME` is typed exactly as how it is spelt on
-  the card.
-* For the **List** commands, to view list of persons in a particular subgroup, you should first use **List** command to list out all the subgroups in a `Group` you want to view,
-  then list out the persons inside that subgroup by **List** command with the `GROUP_INDEX` you want to list out.
-
-## FAQ
-
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains
-the data of your previous Notor home folder.
+Refer to the details of Person commands [here](#person).
```

Comment:
This goes to here literally. It goes to the summary. I think it is suppose to go up to the commands.